<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IceBox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTreeWidget" name="treeWidgetIceBox">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;View added or generated files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string>IceBox Viewer</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="statusEdit">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="iceBoxTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Install </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxIceStrom">
          <property name="text">
           <string>IceStorm</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxArachne">
          <property name="text">
           <string>Arachne-PNR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxYosys">
          <property name="text">
           <string>Yosys</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="inputMask">
           <string extracomment="ps"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Provide User Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonInstall">
          <property name="text">
           <string>Install Software</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarInstall">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Project</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Add/Remove Constraints (.pcf) File</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButtonAddPcf">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a constraints file for your FPGA &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Add *.pcf File</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButtonRemovePcf">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove constraints File&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Remove *.pcf File</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEditProjectName">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Provide Project Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Project Name</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>RTL </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Set Top Level module </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTopModule">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name of top module name in RTL file(s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="placeholderText">
           <string>Top Level Module Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Add/Remove Verilog RTL File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddVerilog">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press Ctrl in the file dialog to Add multiple RTL files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Add Verilog(*.v) File(s)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRemoveVerilog">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Press Ctrl in the file dialog to Remove multiple RTL files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Remove Verilog(*.v) File(s)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Generate</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Select FPGA Part</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxFpgaPart">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select your FPGA package&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Run Synthesis &amp; PNR To Generate FPGA Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGenerateFPGA">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run icestorm flow to generate FPGA image &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Generate FPGA Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarGenerateFPGA">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Pro&amp;ject</string>
    </property>
    <addaction name="actionLoadProject"/>
    <addaction name="actionClearProject"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelpHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuHelpHelp"/>
  </widget>
  <action name="actionCreate_PRoject">
   <property name="text">
    <string>&amp;Create Project</string>
   </property>
  </action>
  <action name="actionLoadProject">
   <property name="text">
    <string>&amp;Load Project</string>
   </property>
  </action>
  <action name="actionClearProject">
   <property name="text">
    <string>&amp;Clear Project</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;View files added to this project&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
